<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Enroll</class>
 <widget class="QWidget" name="Enroll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{
	border-style:none;
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>120</y>
     <width>500</width>
     <height>340</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0);</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QLineEdit" name="password">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>长度6位以上</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordlabel">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="phonenumber">
       <property name="placeholderText">
        <string>11位手机号码</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="checkpassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>请再次输入密码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name">
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="phonelabel">
       <property name="text">
        <string>手机号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="checkpasswordlabel">
       <property name="text">
        <string>确认密码</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="email"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="namelabel">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="emaillabel">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="putin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font:  20pt &quot;微软雅黑&quot;;background-color:rgb(60, 230, 0);border-radius: 3px; color: rgb(255,255,255);}QPushButton:hover { background-color:rgb(40, 220, 0); }

</string>
    </property>
    <property name="text">
     <string>立即注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_check">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>pwd_check</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_check">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>63</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>name_check</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_check">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>102</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>email_check</string>
    </property>
   </widget>
   <widget class="QLabel" name="phone_check">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>phone_check</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>490</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.75, stop:0 rgb(0, 170, 255), stop:1 rgb(255, 255, 255));</string>
   </property>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:transparent;border-radius: 3px; color: rgb(255, 255, 255);}
QPushButton:hover { background-color: rgb(255, 100, 100); }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMovie">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bradley Hand ITC</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 255, 255);
font: Bold 25pt &quot;Bradley Hand ITC&quot;;background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>QuickTalk</string>
    </property>
   </widget>
   <zorder>labelMovie</zorder>
   <zorder>close_btn</zorder>
   <zorder>label</zorder>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
